### DISCUSSION TITLE
Universes without a finite age, heat death, and hydrogen from iron.

### DISCUSSION THREAD

**Discussion by mikehelland** [a-cosmology-group/acg#200] [Astrophysics and Physics]
One of the big problems alternative cosmologies have is that, while the big bang ~14 billion years ago, and inevitable (and also not all that far in the future) heat death seems anti-Copernican in the sense that "this was all done for us", there's also the real physics of thermodynamics that an "eternal" universe has to have an answer for.

And a big part of that is, if everything "goes to iron" so to speak, and it's basically eternal... how would an indefinitely old universe not end up following the same path?

Please, post your ideas on the topic. In the meantime, here's what I've been kicking around the last couple days.

----

Within the first microseconds of the big bang the universe went from a "quark-gluon plasma" to having protons and neutrons:

>Gradually, the Universe undergoes a critical transition: from free up, down, anti-up, and anti-down quarks to bound protons, neutrons, antiprotons, and antineutrons. What was once a quark-gluon plasma has now passed through what’s known as the era of confinement, as temperatures and densities become low enough that free, unbound quark states can no longer stably persist. 

[https://bigthink.com/starts-with-a-b...utrons-formed/](https://bigthink.com/starts-with-a-bang/protons-and-neutrons-formed/)

We can create this quark-gluon plasma in our labs:

>Production of QGP in the laboratory is achieved by colliding heavy atomic nuclei (called heavy ions as in an accelerator atoms are ionized) at relativistic energy in which matter is heated well above the Hagedorn temperature TH = 150 MeV per particle, which amounts to a temperature exceeding 1.66×1012 K. This can be accomplished by colliding two large nuclei at high energy (note that 175 MeV is not the energy of the colliding beam). Lead and gold nuclei have been used for such collisions at CERN SPS and BNL RHIC, respectively. 

[https://en.wikipedia.org/wiki/Quark%...sma#Production](https://en.wikipedia.org/wiki/Quark%E2%80%93gluon_plasma#Production)

If we create QGP from colliding lead, what happens of the lead nuclei? Are they no longer lead? Does the QCP in the lab do what it would do in the big bang, and concentrate into protons?

If we can do it in a lab, does it happen in nature?

It seems quasars and AGN get hundreds of billions of Kelvin:

I just watched this great video on AGN/quasars, blasars, pulsars:

https://www.youtube.com/watch?v=AUeTOE3FtVI

What if heavy nuclei hit stuff while spinning in the super hot accretion disk, which is basically acting as a natural particle accelerator/collidor. Couldn't that break them apart (boil them)? Their constituents become part of the plasma jets blowing away from the black hole?

I realize that's a pile of speculation on a foundation of deep ignorance. But it seems like those plasma jets the big AGN got could form hydrogen nuclei from heavier elements. What happens when the plasma cools? 

----

Looking forward to your comments!

**Comment by RedshiftDrift** [comment]
The "everything goes to iron" argument doesn't work, the energy from light gets used to recycle iron back to hydrogen.  See this discussion [https://github.com/orgs/a-cosmology-group/discussions/144#discussioncomment-7719752](https://github.com/orgs/a-cosmology-group/discussions/144#discussioncomment-7719752)

**Comment by Francois-Zinserling** [comment]
This must be my favourite topic. When I see a topic about quarks, I develop an uncontrolled twitch twitch twitch ...
The story of quarks reads like a fantasy tale, but it is also very telling about the state of particle physics. Yes, the mess is not just in cosmology. Here is the history (my version) of quarks ...

Charges
- Invent 1/3 and 2/3 fractional charges, (of both + and - sign) because the Standard Model does not allow electron to be 'inside' a neutron. Because ... rules. So let's make a new rule that says the electron decays into different particles. All is ok again.

Spins
- The neutron spin should have been zero according to Standard Model rules (0 charge) but we measured a strong magnetic moment. So it must have spin! To balance the spin equations (proton and neutron are now both spin=1/2) we need to assign 'up' and 'down' spins to these fractional charges. Admittedly the charges and spins works out beautifully and this is about as far as I admire about this whole quark expedition. Attempts at detecting quark spins almost immediately led to a 'proton spin crisis' which still persists.

Confinement
- All attempts to knock out a quark from the proton or neutron fail. All the while + and - and ++ and -- and 0 and 00 come flying out. Quarks must be contained even stronger than the strong force? Make a new law of quark confinement which says they cannot exist in isolation and everything is ok again. Now we can't see quarks because there is a law against it! 

Colour Charges
- The quark spins appear to violate Pauli's exclusion principle (2 x 1/2 spin particles with same quantum number in proton or neutron not allowed) so let's add 3 possible colour charges. Oh, and 3 anti-colour charges. New quantum properties specially and only for quarks which also helps to explain the 'confinement' law.

Valence
- We try to measure the masses of the quarks but fail dismally to explain the mass difference between proton and neutron. So let's fix this and say there are more quark-antiquark pairs (LOTS OF THEM) and the UUD and UDD that we need will only be '3 valence quarks'. Something new again.

Gluons
- We still don't get the masses right. In fact we can hardly detect any mass on the quarks, and still cannot explain how this whole mess manages to stay in the proton and neutron 'bag' in such very exact proportions as we got to know it, so we need something that sounds like it will keep it together.... Gluons! Another new thing! But this is ok because gluons can be a new type of particle. They're actually photons, but let's also give them mass and spin =1, so they're not really photons.

A sea of quarks and gluons confined in a bag
This is where we are at present, and somehow from this, all protons manage to be exactly alike.
We have now invented a whole dictionary of new rules and laws, just so that we don't have to give up on fractional charges .... Who were Gell-Mann and Zweig that they held such divinity over physics!?

To this point we have not seen or measured any of these - no single quarks, fractional charges, colour charges, or gluons. After all this we are still not any nearer to a solution, and we have lost about 50 years meddling with this nonsense.

**Reply by HanDeBruijn** [reply]
>When I see a topic about [quark](https://en.wikipedia.org/wiki/Quark_(dairy_product))s [ .. ]

Can't resist:
![kwark](https://github.com/a-cosmology-group/acg/assets/135433222/e4efd35a-66f0-4635-94d8-14f164a28f2e)
>the Standard Model does not allow electron to be 'inside' a neutron.

That would be a simple picture: a neutron is an electron mixed with a proton, or glued on a proton.
>Now we can't see quarks because there is a law against it!

Ah, quarks are like gnomes. Every time you want to see them they are quickly hiding behind a tree. [Charm](https://en.wikipedia.org/wiki/Charm_quark)ing, isn't it? [Strangeness](https://en.wikipedia.org/wiki/Strangeness) too is a very much conserved quantity in contemporary physics. Professor [Poppema](https://scite.ai/authors/o-j-poppema-M2kA2p) started to talk about it and I decided _not_ to become a nuclear physicist.
>We still don't get the masses right.

I have a book in my shelves. It is titled [_Ruimte, Tijd, Materie_](https://www.bol.com/nl/nl/p/epsilon-uitgaven-95-ruimte-tijd-materie/9300000001197011/), (_Space, Time, Matter_), written by Roel Andringa-Boxum, 548 pages.
I had a [discussion](http://www.sterrenstof.info/overdenkingen-in-tijden-van-corona/comment-page-8/#comment-295548) with the author (nickname _Haushofer_) in May 2020 about page 389 in the book: _Deze proton- en neutronmassa uitrekenen aan de hand van hun quarks blijkt nog een hels karwei te zijn waarvoor we supercomputers nodig hebben._ Google translate: _Calculating these proton and neutron masses based on their quarks turns out to be a hellish job for which we need supercomputers._ Yes of course: **brains off, computers on.**
Anyway, Roel [came up](http://www.sterrenstof.info/overdenkingen-in-tijden-van-corona/comment-page-8/#comment-295570) with a paper: [Nuclear masses calculated from scratch](https://www.nature.com/articles/news.2008.1246), accessible via the institution that I am not a member of. It says: _An exhaustive calculation of proton and neutron masses vindicates the Standard Model._ Honestly, we didn't expect otherwise.

**Reply by mikehelland** [reply]
I don't think it's a good look for alternative cosmologies to have to bend the laws of physics in other fields.

That said. It seems AGN and their accretion disks plus jets have the temperature it takes to turn any element into the constituents of hydrogen.

**Comment by HanDeBruijn** [comment]
>there's also the real physics of thermodynamics that an "eternal" universe has to have an answer for.

Do you mean that the entropy of the universe is increasing? So that the universe will end into complete chaos? Then I must disappoint you: **there is no end** that could be caused by such a thing as the increasing entropy of the universe. **Entropy** for open thermodynamical systems is simply **undefined**. Here is a reference, titled [_Adventures in Theoretical Physics I – Entropy_](https://thisislanduniverse.com/2021/05/30/adventures-in-theoretical-physics-i-entropy/) written by an ACG member. And we have a video by Sabine Hossenfelder in [thread 62](https://github.com/orgs/a-cosmology-group/discussions/62) .

**Comment by ExpEarth** [comment]
> One of the big problems alternative cosmologies have is that, while the big bang ~14 billion years ago, and inevitable (and also not all that far in the future) heat death seems anti-Copernican in the sense that "this was all done for us", there's also the real physics of thermodynamics that an "eternal" universe has to have an answer for.

The heat death is not necessary, but for that you need to have a different model of spacetime than what is usually given.  It has to have a photonic structure, such as the overlapping double photon model of Annila et al:

![image](https://github.com/a-cosmology-group/acg/assets/136624758/c21462d4-dec5-4f32-bc20-b234feb847b1)

With such a model you can explain many things.  First, the Hubble redshift itself involves transfer of photon energy to spacetime.  Such a Hubble redshift can then account for the temperature-distance relationship in the CMB in a satisfactory manner.   Most significantly, photon energy in the universe never gets depleted, as spacetime gives back energy to photons of equal magnitude to the Hubble redshift losses.   These 'Hubble blueshifts' can also account for the cosmological constant, in Einstein's sense.  The universe is a quasi-permanent structure.  It can grow through accretion or shrink by losing mass, but it goes on.

**Reply by Francois-Zinserling** [reply]
@ExpEarth 
1. "_redshift itself involves transfer of photon energy to spacetime_"
2. "_spacetime gives back energy to photons of equal magnitude to the Hubble redshift losses_"

Are these mechanisms explained by Arto Anilla? Do you know where they propose these actions to physically occur? 
How does your own model manifest these actions?

My conceptual aether model ( I also referenced Patrick Grahn, Arto Annila, Erkki Kolehmainen) is based on a sea of mutually destructive interfering spin=1 bosons, which presents itself as a boson spin = 2 (or spin=0?) tensor field, mostly undetectable because of the 'self-interfering' and leads nicely to understanding the concept of inertia. With a little added effort, it could answer to point 1 above. 
One shortcoming of my idea is the (unknown) source of these bosons/photons. (point 2 above)

**Reply by ExpEarth** [reply]
@Francois-Zinserling 

Thanks for the interesting post!  The mechanisms 1 and 2 are not described by Arto Annila et al.  They are my own.  My model is like theirs (I use several parts of theirs), but they miss a key feature.  The filaments of gravitons (made of photons) have to be attached to masses for my gravity mechanism to work!  They imagine gravity to happen in a different way and they also suppose that the universe is expanding.

The Hubble redshift in my model involves transfer of photon energy to these filaments (which I call spacetime) and the Hubble luminosity reverses this process, generating new photonic energy.  The reasons for these transfers have to do with the temperature of the CMB and the equivalent 'temperature' of spacetime in my model.  It's a flow of photon energy from hotter to cooler regions.

Your model sounds quite similar to Arto's model and it seems you have extended it beyond what they have done.  Very good!  Instead of saying spin 1 bosons and spin 2 bosons, you could just say photons and gravitons.  Eventually your question relates to where the CMB came from.  The CMB temperature calibrates spacetime 'temperature' in my model.

**Comment by HanDeBruijn** [comment]
>heat death

All human religions talk about **eternal life** but our so-called "science" manages to talk about a **heat death** of everything.
Indeed. There is a real danger that our "civilization" will end in heat death. We have enough (nuclear) weapons to accomplish it.
That says nothing about the universe. It tells only something about ourselves.

**Comment by mikehelland** [comment]
I put together a little video on this topic, Do quasar jets produce fresh hydrogen?

https://www.youtube.com/watch?v=zbJszZp1yb8

**Reply by HanDeBruijn** [reply]
@mikehelland \: [The ongoing disaster of the standard model of particle physics](https://github.com/orgs/a-cosmology-group/discussions/201) .

**Reply by mikehelland** [reply]
That doesn't seem relevant. All that we need to know has been shown in labs. Hadrons boil at 1.6 trillion K.

**Reply by RedshiftDrift** [reply]
I do not think that quasars are what the standard model describes, and there are [no consistent observations of the jets moving away from the bright galaxy](https://github.com/orgs/a-cosmology-group/discussions/144#discussioncomment-7949750).

Hydrogen [is produced by fission in the IGM](https://github.com/orgs/a-cosmology-group/discussions/144#discussioncomment-7719752)

**Reply by mikehelland** [reply]
It could be they are something different, but this really close by quasar has been measured to have temps that exceed the known boiling point of nuclei. So at least based on the known-knowns, here seems to be a source of hydrogen.

**Reply by HanDeBruijn** [reply]
What follows is loosely related to the (_very informative_, thanks) [video](https://www.youtube.com/watch?v=zbJszZp1yb8) by @mikehelland .

## [Planck units](https://en.wikipedia.org/wiki/Planck_units)

Basic hypothesis: **[(Micro) black hole](https://en.wikipedia.org/wiki/Micro_black_hole)s do not exist**.
As a consequence: the "radius" of an elementary particle is always greater than its [Schwarzschild radius](https://en.wikipedia.org/wiki/Schwarzschild_radius) .
Where the "radius" is assumed to be the [Compton wavelength](https://en.wikipedia.org/wiki/Compton_wavelength) $\lambda_e$ of the particle.
Mathematically, with $m_e=$ elementary particle mass, $\hbar=$ [Reduced Planck constant](https://en.wikipedia.org/wiki/Planck_constant#Reduced_Planck_constant), $c=$ speed of light, $G=$ gravitational constant (and forget about a factor $2$) :

$$
\lambda_e = \frac{\hbar}{m_e.c} \gt \frac{G.m_e}{c^2} \quad \Longrightarrow \quad m_e \lt \sqrt{\hbar c/G}
$$

Let the _Planck mass_ $m_P$ be defined as

$$
m_P = \sqrt{\hbar c/G} \quad \Longrightarrow \quad m_e \lt m_P
$$

Ipse est (i.e.): an elementary particle mass is always smaller than the Planck mass.
The Planck mass is an absolute upper bound for the mass of an elementary particle.

For the Compton wavelength $\lambda_e$ of an elementary particle we have

$$
\lambda_e = \frac{\hbar}{m_e.c} \gt \frac{\hbar}{m_P.c} \quad \Longrightarrow \quad \lambda_e \gt \sqrt{\hbar G/c^3} = l_P
$$

Consequently, the "size" $\lambda_e$ of an elementary particle is always greater than the Planck length $l_P$ .
Associated with the Compton wavelength is a Compton time, which is greater than the Planck time $t_P$ .

$$
\frac{\lambda_e}{c} \gt \sqrt{\hbar G/c^5} = t_P
$$

Additional information:  [The Limited Validity of Special Relativity](http://www.alternatievewiskunde.nl/einstein/AanDavid.htm) .

Last but not least we have the **Planck temperature**. Two quotes (found with Google):
_The Planck temperature is the fundamental upper limit of temperature_.
_At this temperature, the wavelength of light emitted by thermal radiation reaches the Planck length_.
The latter can more or less be derived with [Wien's displacement law](https://en.wikipedia.org/wiki/Wien%27s_displacement_law#Derivation_from_Planck's_law), where $k_B=$ [Boltzmann constant](https://en.wikipedia.org/wiki/Boltzmann_constant), $T=$ temperature.

$$
\frac{h c}{\lambda_{peak}} = x k_B T \quad \mbox{with} \quad x = 4.965114231744276304
$$

Apart from less important math constants we thus derive that for any temperature

$$
\frac{\hbar c}{\sqrt{\hbar G/c^3}} \gt k_B T \quad \Longrightarrow \quad T \lt \sqrt{\hbar c^5/(G k_B^2)}
$$

Calculations:
```
hbar := 1.054571817*10^(-34);
c := 299792458;
G := 6.674*10^(-11);
l_P := sqrt(hbar*G/c^3);
m_P := sqrt(hbar*c/G);
t_P := sqrt(hbar*G/c^5);
k_B := 1.380649*10^(-23);
T_P := sqrt(hbar*c^5/(G*k_B^2));
```
Summarizing \:

$$
\begin{matrix}
\mbox{Planck mass} & \: & m_P=\sqrt{\hbar c/G} & \approx 0.2176483258 \times 10^{-7}\mbox{ }kg \\
\mbox{Planck length} & \: & l_P=\sqrt{\hbar G/c^3} & \approx 0.1616218699 \times 10^{-34}\mbox{ }m \\
\mbox{Planck time} & \: & t_P=\sqrt{\hbar G/c^5} & \approx 0.5391125281 \times 10^{-43}\mbox{ }s \\
\mbox{Planck temperature} & \: & T_P=\sqrt{\hbar c^5/(G k_B^2)} & \approx 0.1416816004 \times 10^{33}\mbox{ }K \\
\end{matrix}
$$

The [Hagedorn temperature](https://en.wikipedia.org/wiki/Hagedorn_temperature) in the video is still orders of magnitude less than he Planck temperature.
